File: MODUL 6 PEMROGRAMAN 1/PRAK604-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[100];  
    char pesan[100];   

    fgets(kode, sizeof(kode), stdin);
    fgets(pesan, sizeof(pesan), stdin);

    kode[strcspn(kode, "\n")] = '\0';
    pesan[strcspn(pesan, "\n")] = '\0';

    int length = strlen(kode);
    int length2 = strlen(pesan);
    int bintang = 0;
    int pagar = 0;
        
        if (length == length2){
            for (int i = 0; i < length; i++) {
                if (kode[i] != pesan[i]) {
                    printf("#");
                    pagar++;
                } else if (kode[i] == ' ' && pesan[i] == ' ') {
                    printf(" ");
                } else {
                    printf("*"); 
                    bintang++;
                }
            }printf("\n* = %d\n", bintang);
            printf("# = %d\n", pagar);
        
            if (bintang >= pagar) {
                printf("Pesan Asli\n");
            } else {
                printf("Pesan Palsu\n");
            }
        }else{
            printf("Panjang kalimat berbeda, pesan palsu\n");
        }
    return 0;
}

File: MODUL 6 PEMROGRAMAN 1/PRAK604-2310817120002-SITIRATNADWINTASARI.py
kode = input().rstrip()
pesan = input().rstrip()

length = len(kode)
length2 = len(pesan)
bintang = 0
pagar = 0

if length == length2:
    for i in range(length):
        if kode[i] != pesan[i]:
            print("#", end="")
            pagar += 1
        elif kode[i] == ' ' and pesan[i] == ' ':
            print(" ", end="")
        else:
            print("*", end="")
            bintang += 1

    print("\n* =", bintang)
    print("# =", pagar)

    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
else:
    print("Panjang kalimat berbeda, pesan palsu")
